kolibri/plugins/learn/assets/src/composables/useDevices.js

```diff
@@ -11,13 +11,15 @@ import plugin_data from 'plugin_data';
 import { KolibriStudioId } from '../constants';
 import { learnStrings } from '../views/commonLearnStrings';
 
-// The refs are defined in the outer scope so they can be used as a shared store
+/**
+ * The ref is defined in the outer scope so it can be used as a shared store
+ * @type {Ref<NetworkLocation|null>}
+ */
 const currentDevice = ref(null);
 
 const KolibriStudioDeviceData = {
+  ...plugin_data.studioDevice,
```

Comment:
If we didn't filter the Reserved locations out of the NetworkLocations API response, could we just be getting most of this data from the API instead now? (we'd still need to translate the name, and filter it for appropriate learner types).

Reply:
Yes, very likely. I was unsure of the implications of preventing the reserved Studio instance from the connection testing, and how that would be handled by consuming usages, so I decided to leave that to the scope of the other outstanding issue